frontend/src/components/StocksYouFollow/StocksYouFollow.jsx

```diff
@@ -39,30 +45,78 @@ export default function StocksYouFollow({ stocks, setStocks }) {
-            </div>
+            <Table
+                className="rounded-table"
+                striped
+                bordered
+                hover
+                variant="dark"
+            >
+                <thead>
+                    <tr>
+                        <th className="text-center">Logo</th>
+                        <th className="text-center">Ticker</th>
+                        <th className="text-center">Price</th>
+                        <th className="text-center">Sector</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {stocks
+                        .slice(0, Math.min(displayedStocks, stocks.length))
+                        .map((stock) => {
+                            return (
+                                <tr key={stock.ticker}>
+                                    <td className="stock-image">
+                                        <div className="stock-follow-logo">
+                                            <img
+                                                src={stock.logo}
+                                                alt={`This is a logo of ${stock.name}.`}
+                                                onLoad={() =>
+                                                    setLoadedLogos((prev) => ({
+                                                        ...prev,
+                                                        [stock.ticker]: true
+                                                    }))
+                                                }
+                                                style={{
+                                                    visibility: loadedLogos[
+                                                        stock.ticker
+                                                    ]
+                                                        ? "visible"
+                                                        : "hidden"
+                                                }}
+                                            />
+                                        </div>
+                                    </td>
+                                    <td className="text-center">
+                                        <Link
+                                            className="stock-link"
+                                            to={`/search/stocks/${stock.ticker}`}
+                                        >
+                                            {stock.ticker}
+                                        </Link>
+                                    </td>
+                                    <td className="text-center">
+                                        ${stock.price}
+                                    </td>
+                                    <td className="text-center">
+                                        {stock.sector}
+                                    </td>
+                                </tr>
```

Comment:
I recommend making a new component for this, as there's a lot of logic happening in the table. Something like `StocksYouFollowTable` can have a child component of `StockYouFollowRow` (you can think of different component names as well).